### PR TITLE
Support `dbg!` macro

### LINKED ISSUE #3265: No autocompletion or documentation in dbg!() macro
## Environment

* **Intellij-Rust plugin version:**
v0.2.0.2114-183

* **Rust toolchain version:**
Default host: x86_64-pc-windows-msvc
stable-x86_64-pc-windows-msvc (default)
rustc 1.32.0 (9fda7c223 2019-01-16)

* **IDE name and version:**
image

* **Operating system:**
image

## Problem description
In the `dbg!` macro there is no completion or quick documentation. Here is the expected behaviour, in the `println!` macro:
println

Here is the behaviour inside the dbg! macro:
dbg

 

## Steps to reproduce
Write the dbg! macro and stuff inside it...

### PR DESCRIPTION
**Pull request by Undin**
Closes #3265

### CODE REVIEW

File: src/test/kotlin/org/rust/ide/formatter/RsFormatterTest.kt [hunk 1]
```diff
@@ -417,6 +417,18 @@ class RsFormatterTest : RsFormatterTestBase() {
             error!(
             "Hello, World"
             );
+
```
Comment (mchernyavsky) [inline src/test/kotlin/org/rust/ide/formatter/RsFormatterTest.kt]: Maybe we should also add a test for indentation in the `try` macro?

**Comment by Undin** [comment]
bors r=mchernyavsky

**Comment by mchernyavsky** [comment]
bors retry

**Comment by Undin** [comment]
bors r=mchernyavsky

**Pull request merged**